demo-forms/custom_contacts/README.MD

```diff
@@ -0,0 +1,11 @@
+# Applying Sample Custom Contact Types
+
+
+1. Download the current `app_settings.json`
+1. Replace the existing `contact_types` with the contents of [contact_types.json](contact_types.json)
+1. Upload the change to `app_settings.json`
+1. Download the english language file.
+1. Add the translation keys in [en_translations](en_translations.properties)
+1. Upload the translation file. 
```

Comment:
Help 'em out here with the `medic-conf` call to do this.

Reply:
I didn't put `medic-conf` for these operations because the admin interface is nicer to me.

Reply:
Updated with where in the admin app to upload/download.